docs/advanced/chaos-experimentation.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Chaos Experimentation Framework
+{{ .EditURL }}
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+A framework to perform Chaos experiments built on top of [EnvoyProxy](https://www.envoyproxy.io/docs/envoy/latest/)
+
+<img alt="Clutch Component Architecture" src={useBaseUrl('img/docs/chaos-experimentation.png')} width="75%" />
+
+Chaos Experimentation Framework consists of a few parts - frontend, a backend server and a xDS management server. 
+
+The Frontend uses [Clutch's core frontend](https://clutch.sh/docs/development/frontend). It can be customized by using the frontend config. 
+
+The Backend server is responsible for performing CRUD operations of the experimentation package - [CreateExperiments](https://github.com/lyft/clutch/blob/71f84e4bb3f642a17b831019f188a87dcc63f2cf/backend/module/chaos/experimentation/api/experimentation.go#L68), [GetExperiments](https://github.com/lyft/clutch/blob/71f84e4bb3f642a17b831019f188a87dcc63f2cf/backend/module/chaos/experimentation/api/experimentation.go#L134), [CancelExperimentRun](https://github.com/lyft/clutch/blob/71f84e4bb3f642a17b831019f188a87dcc63f2cf/backend/module/chaos/experimentation/api/experimentation.go#L123), etc. It stores experiments in its tables in the Postgres database. 
+
+The xDS management server uses [go-control-plane](http://github.com/envoyproxy/go-control-plane) library and serves two Envoy APIs - [Runtime Discovery Service (RTDS)](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/operations/dynamic_configuration#rtds) and [Extension Configuration Discovery Service (ECDS)](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/operations/dynamic_configuration#ecds). Either of these two APIs can be used to perform fault injection tests. With RTDS, you can make changes to runtime specific to faults whereas with ECDS you can make changes to the entire fault filter to perform any custom experiments.
+
+### Components
+
+Below components are responsible to perform experiments starting from storing the data into the Postgres database for each incoming request all the way to passing the experiment values to the Envoys to inject faults.
```

Comment:
```suggestion
Below components are responsible to perform chaos experiments starting from storing the data into the Postgres database for each incoming request all the way to passing the experiment values to the Envoys to inject faults.
```

let's start by using "chaos experiments" phrase in a few places so that a reader understands what kind of experiments are we talking about later on.

Reply:
I don't think we need to add `chaos` everywhere. The document heading itself has `Chaos` in it. If one reads the doc from the beginning it's understood that it's talking about `chaos` experiments. 

If you still feel strongly, happy to commit all the suggestions you have. Let me know

Reply:
I do agree that it won't hurt to have the word `chaos` sprinkled in more places. If anything it will help with SEO

Reply:
Added `chaos` as people feel strongly about it :)